src/main/java/core/basesyntax/model/machine/Machine.java

```diff
@@ -1,9 +1,27 @@
 package core.basesyntax.model.machine;
 
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Inheritance;
+import javax.persistence.InheritanceType;
+
+@Entity
+@Inheritance(strategy = InheritanceType.TABLE_PER_CLASS)
 public class Machine {
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
```

Comment:
with this inheritance strategy we need a different id generation type
also, `@ID` annotation is missing

Reply:
Id added. Straregy shanges on "TABLE"